### PR TITLE
Add price in eth metric to API

### PR DESCRIPTION
**Pull request by antisaa**
## Changes

Adding new metrics to API
- price in eth
- labeled holders metrics
## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have tried to find clearer solution before commenting hard-to-understand parts of code
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Review by tspenov — APPROVED** [review]

**Comment by tspenov** [comment]
Although we have to remove the current `price_eth` before merging

**Comment by antisaa** [comment]
> Although we have to remove the current `price_eth` before merging

I can't find `price_eth` in available metrics, how is it being computed?

**Comment by tspenov** [comment]
The timeseries metric is here: https://github.com/santiment/sanbase2/blob/master/lib/sanbase/prices/price.ex#L173
It now takes the price of asset in usd and the price of eth in usd and divides them.
I'll take care of removing and testing it. It should be also replaced per project with the new metric.

**Comment by tspenov** [comment]
The tests are failing because you are using the `name`field  instead of `metric` field in tests. @antisaa

**Comment by tspenov** [comment]
@antisaa my bad. The .json file was missing from the file handler. Pushed a fix.

File: lib/sanbase/clickhouse/metric/metric_files/available_v2_metrics.json [hunk 1]
```diff
@@ -2081,5 +2081,22 @@
...
+    "access": "free",
+    "selectors": ["slug"],
+    "min_plan": {
+      "SANAPI": "free",
+      "SANBASE": "free"
+    },
+    "aggregation": "last",
+    "min_interval": "5m",
+    "table": "intraday_metrics",
+    "has_incomplete_data": true,
```
Comment (IvanIvanoff) [inline lib/sanbase/clickhouse/metric/metric_files/available_v2_metrics.json:2099]: This should be `false`

**Pull request merged**